azext_iot/operations/hub.py

```diff
@@ -2221,169 +2221,159 @@ def iot_get_module_connection_string(
     return result
 
 
-def iot_device_export(
+def _get_service_sdk(
```

Comment:
This _get_service_sdk implementation is limited where it doesn't support `login` and `auth_type_dataplane` parameters. See the legacy pattern in how the plumbing works  https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/operations/hub.py#L172.